app/views/records/new.html.erb

```diff
@@ -0,0 +1,33 @@
+ 
+  <%= form_with model: @category_records, url:category_records_path, scope: :record, method: :post, data: { turbo: false }, class: "form" do |form| %>
+    <div class="nav">
+      <%= link_to category_records_path do %>
+        <i class="fa-solid fa-arrow-left back"></i>
+      <% end %>
+      <div>
+        ADD A NEW TRANSACTION
+      </div>
+      <div class="actions">
+        <%= form.submit "ADD", class: "loginBtn" %>
+      </div>
+    </div>
+    <div class="textField">
+      <div class="field">
+        <%= form.text_field :name, placeholder: "Transaction name", autofocus: true, autocomplete: "name", class: "form-control stuff" %>
+      </div>
+
+      <div class="field">
+        <%= form.text_field :amount, placeholder: "Amount", autofocus: true, autocomplete: "email", class: "form-control stuff" %>
+      </div>
+
+      <div class="card checkbox">
+        <%= form.label :categories, 'Select a Category', class: 'title' %><br>
+        <%= form.collection_check_boxes :category_ids, Category.where(user_id: current_user.id), :id, :name, class: '' do |cat| %>
+          <div class="checkbox-div">
+            <%= cat.check_box %>
+            <%= cat.label class: 'checkbox' %><br>
+          </div>
+        <% end %>
+      </div>
```

Comment:
- [ ] **[OPTIONAL]** It would be nice to add the `required` attribute here as well